### DISCUSSION TITLE
FMU Export Fails for Borefield Examples in Buildings and IDEAS Libraries

### DISCUSSION THREAD

**Discussion by Taubenstrohhalm** [OpenModelica/OpenModelica#13390] [Q&A] [answered]
When attempting to export FMUs for the Borefield examples in both the Buildings and IDEAS libraries, the process fails with an error related to LAPACK:
```
[100%] Linking C shared library Borefields.dll 
C:/Program Files/OpenModelica1.24.2-64bit/tools/msys/ucrt64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: CMakeFiles/Borefields.dir/objects.a(Borefields_functions.c.obj): in function `omc_Modelica_Math_Matrices_LAPACK_dgetri': 
C:/Users/USER.NAME/AppData/Local/Temp/OpenModelica/OMEdit/Borefields2f01/Borefields.fmutmp/sources/Borefields_functions.c:12509:(.text+0x2d038): undefined reference to `dgetri_' 
collect2.exe: error: ld returned 1 exit status
```

**Steps to Reproduce:**
Load the Buildings or IDEAS library in OpenModelica.
Open any Borefield example model.
Attempt to export it as an FMU using either GCC or Clang as the compiler.

**Observed Behavior:**
FMU export fails with the error above. The issue seems to arise due to a missing LAPACK dependency (dgetri_). This behavior is consistent regardless of whether GCC or Clang is used as the compiler.

**Expected Behavior:**
FMU export should complete successfully without errors.

**Environment:**
OpenModelica version: 1.24.2
Operating System: Windows 11
Compiler: GCC and Clang (both tested)
Buildings library version: 11.0.0
IDEAS library version: 3.0.0

**Questions:**
1. Is this a known issue or a bug in OpenModelica’s FMU export process for models with dependencies on LAPACK?
2. Are there any specific settings I need to configure for exporting these FMUs?

**Comment by tinoetzold** [comment, marked as answer]
Hi, 
I faced the same issue. There are some reported issues, e.g.

https://github.com/OpenModelica/OpenModelica/issues/13719

Regards
Tino
